catalog.tests.bom

```diff
@@ -0,0 +1,127 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+brooklyn.catalog:
+  items:
+  - id: jboss-application-server-6-tests
+    version: 1.0
+    itemType: template
+    iconUrl: https://github.com/brooklyncentral/brooklyn-jboss-application-server-6/raw/master/jboss-logo.png
+    name: JBoss AS 6 tests
```

Comment:
The catalog item is called `JBoss Application Server 6`. Maybe the test should follow the same convention, i.e `JBoss Application Server 6 tests`?